package.json

```diff
@@ -99,7 +100,7 @@
     "highlight.js": "^9.3.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": ">=0.14.0 || <=16.0.0",
+    "react-dom": ">=0.14.0 || <=16.0.0"
```

Comment:
We're getting ready to release react@16.1.0. (We released a beta just last week.) I don't want to update versions only to lock people in at 16.0.0. I think a better version patter would be: `^0.14.0 || ^15.0.0 || ^16.0.0`

Reply:
Oh, crap, you are right, make sense. Thank you!

Reply:
No worries 😄